src/plugins/stats.ts

```diff
@@ -1,9 +1,9 @@
 // Built-in plugin to expose stats about the deployment
-module.exports = async robot => {
+module.exports = async (robot: any): Promise<void> => {
   const REFRESH_INTERVAL = 60 * 60 * 1000
 
   // Cache of stats that get reported
-  const stats = {installations: 0, popular: []}
+  let stats = {installations: 0, popular: [{}]}
```

Comment:
why are you changing this to let?

Reply:
I think I was fighting with the `never[]` type and did this out of confusion. Will revert this.